app/models/label/LabelTable.scala

```diff
@@ -1038,8 +1038,8 @@ object LabelTable {
       if _l.userId === userId.toString
       if regionId.isEmpty.asColumnOf[Boolean] || _ser.regionId === regionId.getOrElse(-1)
       if _lp.lat.isDefined && _lp.lng.isDefined
-    } yield (_l.labelId, _l.auditTaskId, _l.gsvPanoramaId, _lt.labelType, _lp.lat.get, _lp.lng.get)
-    _labels.list.map(LabelLocation.tupled)
+    } yield (_l.labelId, _l.auditTaskId, _l.gsvPanoramaId, _lt.labelType, _lp.lat, _lp.lng, _l.correct, _l.agreeCount > 0 || _l.disagreeCount > 0 || _l.unsureCount > 0)
+    _labels.list.map(l => LabelLocation(l._1, l._2, l._3, l._4, l._5.get, l._6.get, l._7, l._8))
```

Comment:
Is there a reason why `.tupled` no longer works here?

Reply:
There was a typing issue with the previous implementation that also came up previously (see line 1022-1025 in LabelTable.scala, selectLocationsAndSeveritiesOfLabels).

Reply:
Got it, thank you. Forgot about that!